src/trade/tradeDialog.es6

```diff
@@ -379,8 +379,8 @@ function init_state(available,root, dialog, symbol, contracts_for_spot){
       name: '',
       visible: false,
     },
+    basis_decimals: isCryptoCurrency(local_storage.get('currency')) ? 8 : 2,
```

Comment:
is the crypto decimals always 8 ?

Reply:
instead of adding new object, we can put it under `basis` object on line 296 ?, so basis.decimals